api/editor/checkstyles.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
Attempting to add linting rules.  They now are displayed in the IDE, but it's impossible to manually or automatically fix the issues...  Driving me crazy.